apps/coscrad-frontend-e2e/src/e2e/resources/terms/term.query-flow.e2e.cy.ts

```diff
@@ -63,6 +63,98 @@ describe(`Term index-to-detail flow`, () => {
 
             cy.location('pathname').should('contain', `/Resources/Terms/${basicTermId}`);
         });
+
+        describe.only(`the search filter`, () => {
+            const termWithQDash = 'Q-Term';
+
+            const dummyEnglishTranslationOfTerm = `ZZZ Term (English)`;
+
+            const searchScopes = [`allProperties`, `name`];
+
+            before(() => {
+                cy.seedDataWithCommand(`CREATE_TERM`, {
+                    aggregateCompositeIdentifier: buildDummyAggregateCompositeIdentifier(
+                        AggregateType.term,
+                        516
+                    ),
+                    text: termWithQDash,
+                    languageCode: LanguageCode.Haida,
+                });
+
+                cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+                    aggregateCompositeIdentifier: buildDummyAggregateCompositeIdentifier(
+                        AggregateType.term,
+                        516
+                    ),
+                });
+
+                cy.seedDataWithCommand(`TRANSLATE_TERM`, {
+                    aggregateCompositeIdentifier: buildDummyAggregateCompositeIdentifier(
+                        AggregateType.term,
+                        516
+                    ),
+                    translation: dummyEnglishTranslationOfTerm,
+                    languageCode: LanguageCode.English,
+                });
+            });
+
+            searchScopes.forEach((searchScope) => {
+                describe(`when the search scope is: ${searchScope}`, () => {
+                    beforeEach(() => {
+                        cy.visit('/Resources/Terms');
+
+                        cy.getByDataAttribute('select_index_search_scope')
+                            .click()
+                            .get(`[data-value="${searchScope}"]`)
+                            .click();
+                    });
+
+                    describe(`when the filter should return 1 result (based on default language term)`, () => {
```

Comment:
Be sure to add a case with 
```js
const searchTerms = `{hai}:${textToFind}`
```

Reply:
I've added this and used the:
`parseSpecialCharSequences: false`
option for `.type()`